ConsentViewController/Classes/GDPRConsentViewController.swift

```diff
@@ -263,7 +265,7 @@ typealias Meta = String
 
     func didAuthIdChange(newAuthId: String?) -> Bool {
         return newAuthId != nil &&
-            localStorage.authId != nil &&
+            localStorage.authId?.count ?? 0 > 0 &&
```

Comment:
@vilasma I think we could use 
```swift
localStorage.authId?.isEmpty ?? true
```

Reply:
@andresilveirah this will always return true instead of that will try with `localStorage.authId?.isEmpty ?? false`